### ISSUE TITLE
508-defect-2 [SCREENREADER] Results screen, screen reader unable to determine search radius.

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#20471]
Label: 14321

    - [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

Issue Title: Results screen, screen reader unable to determine search radius.

## User Story or Problem Statement:
When conducting a search using a screen reader, there is no way to tell how wide an area I am searching. This is particularly problematic in situations where there are no results, when I can't use the search results to estimate the search radius.
Details

## Acceptance Criteria
As a screen reader user, I need some way of knowing the map radius, either on the search results screen or by setting it when conducting my search.

## Environment
JAWS 2021 on a Windows 10 machine using Chrome.
Steps to Recreate

## Proposed Solution (if known)
Is it possible to create a control which intigrates with the mapping software and allows me to set the search radius?

**Comment by mmiddaugh** [comment]
Addressed during #17963
